conf/messages.en

```diff
@@ -494,3 +495,9 @@ rulingInformation.holder = Holder
 rulingInformation.ruling = Ruling reference
 rulingInformation.startDate = Start date
 rulingInformation.expiryDate = Expiry date
+
+applicationView.title = View application - GOV.UK
+applicationView.applicationLink = View application
+applicationView.applicationPDFLink = View application PDF (opens in new window or tab)
+applicationView.whatHappensNext.html = <span id="applicationView.whatHappensNext"><p>HMRC will issue your Binding Tariff Information ruling by email within 30 days unless you:</p><ul class="list list-bullet"><li>are sending samples</li><li>are asked for more information to support your application</li></ul><p>You can find out the status of your application by signing into your account through the guide <a href="https://www.gov.uk/guidance/apply-for-a-binding-tariff-information-decision-notice-600">Apply for a Binding Tariff Information (BTI) ruling (Notice 600), this will open in a new window.</a></p></span>
```

Comment:
This is a changed version of the already existing one a couple of hundred lines above

Reply:
I guess we treat this sections slightly different because initially they only belong to a pdf and we were happy to have some bits there... But now we share this content with the views and probably it makes more sense to split what is content and what is html

Reply:
Yes I agree actually  the html should be in the view and the text in the messages file. Went along with what was already there - but that's a cleaner solution

Reply:
Done